utility/src/main/java/org/oppia/android/util/parser/image/UrlImageParser.kt

```diff
@@ -460,9 +460,13 @@ private fun calculateInitialMargin(availableAreaWidth: Int, drawableWidth: Float
   return margin.coerceAtLeast(0f)
 }
 
-// Reference: https://stackoverflow.com/a/51865494
+// Reference: https://stackoverflow.com/a/51865494 and
+// https://stackoverflow.com/a/35444014/12314934.
 private fun TextView.width(computeWidthOnGlobalLayout: (Int) -> Unit) {
   if (width == 0) {
+    // requestLayout() will request onGlobalLayoutListener when we have images inside recyclerview
+    // with a content description.
```

Comment:
I don't think this quite fully explains the "why." What I think is missing: what about the situation of images in a recycler view leads to the global layout listener approach not working (whereas it works for other situations)? That, I think, is the missing context that we need to detail here.

Reply:
Updated the comment, I hope now it give context.